input/api/rss/index.yaml

```diff
@@ -0,0 +1,5 @@
+FeedRss: true
+FeedSources: "__data/posts/*"
```

Comment:
This needs to filter on `Visible: true` -- or at least exclude `Visible: false`

Reply:
Excellent point - depending on how I add the filter those posts will either already be filtered out _or_ there’s a `FeedFilter` key we can use for exactly that purpose. Either way good catch - I’ll verify before moving out of draft.

Reply:
Verified that the new `Excluded` key I'm adding will filter out any excluded posts since they're removed from the pipeline early.